src/test/run-pass/builtin-clone-unwind.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Care to merge this to `src/test/run-pass/dynamic-drop.rs`, or at least add a test there? It has infrastructure that automatically triggers panics in all places, so we can be more sure you didn't miss an edge case.